environments/demo/demo.tfvars

```diff
@@ -14,6 +14,34 @@ frontends = [
     custom_domain  = "pip-frontend.demo.platform.hmcts.net"
     backend_domain = ["firewall-prod-int-palo-sdsdemo.uksouth.cloudapp.azure.com"]
 
+    disabled_rules = {}
+  },
+  {
+    name           = "vh-test-web"
```

Comment:
don't expect this setup in demo to actually work, it's not setup and not planned like this currently

Reply:
Demo has been removed.